src/en/charms-destroy.md

```diff
@@ -1,112 +1,128 @@
 Title: Removing services, units and environments in Juju
 
-# Removal within Juju
 
-Juju isn't just about magically spinning up services as and when you need them,
-it is also about quickly, sanely and efficiently removing something when you no
-longer need it. This section deals with the sort of things you can ruthlessly
-destroy, and how to go about it.
+# Removing services, units and environments
 
-## Removing Services
+Juju can sanely and efficiently removing something when you no longer need it.
+This section looks at how to remove services, units and environments.
 
-Once a service is no longer required it can be removed with a simple command.
+
+## Removing a service
+
+Once a service is no longer required it can be removed with:
 
 ```bash
-juju remove-service <service-name>
+juju destroy-service <service-name>
 ```
 
-**Warning!:** Removing a service which has active relations with another running service will break that relation. This can cause errors in both services, as such review and potentially remove any relationships first.
+**Warning!:** Removing a service which has active relations with another
```

Comment:
Now would be a good time to update these to the proper admonition style, which is:

!!! Warning:
!!! Note: 

etc